ngraph/core/reference/include/ngraph/runtime/reference/detection_output.hpp

```diff
@@ -7,6 +7,7 @@
 #include <cstddef>
 #include <map>
 #include <string>
+#include <utility>
```

Comment:
Why do we need <utility> here?

Reply:
Comparison operators for std::pair are defined there.

Reply:
Why did it work without it before?

Reply:
Previously only `dataType` was compared, now `T` is also compared. In some calls `T` (type of the second element of the pair) itself is a pair.

Reply:
OK, resolved.